README.md

```diff
@@ -32,6 +32,32 @@ To run the server on the IP address `0.0.0.0`, read-only, on port `1234` in the
 tftpd -i 0.0.0.0 -p 1234 -d "/home/user/tftp" -r
 ```
 
+## Usage (Client)
+
+To install the client and server using Cargo:
+
+```bash
+cargo install --features client tftpd
+tftpd client --help
+tftpd server --help
+```
+
+To run the server on the IP address `0.0.0.0`, read-only, on port `1234` in the `/home/user/tftp` directory:
+
+```bash
+tftpd server -i 0.0.0.0 -p 1234 -d "/home/user/tftp" -r
+```
+
+To connect the client to a tftp server running on IP address `127.0.0.1`, read-only, on port `1234` and download a file named `example.file`
+```bash
+tftpd client example.file -i 0.0.0.0 -p 1234 -d
```

Comment:
I think this would have been cleaner to just add a new binary to the Cargo.toml

```toml
[[bin]]
name = "tftpc"
path = "src/client_main.rs"
required-features = ["client"]

[[bin]]
name = "tftpd"
path = "src/main.rs"
```

Reply:
Would you like to open a new PR for this? We can merge it in before doing a new release.